lib/install/management/appliance.go

```diff
@@ -142,7 +141,7 @@ func (d *Dispatcher) checkExistence(conf *config.VirtualContainerHostConfigSpec,
 		verr = errors.Errorf("Found virtual machine %q, but it is not a VCH. Please choose a different virtual app.", conf.Name)
```

Comment:
This also references a vapp... but I have not changed that since it was not in scope of this ticket.